### ISSUE TITLE
Export Symbol Lookup Error

### ISSUE THREAD

**Issue by lukasmartinelli** [osm2vectortiles/osm2vectortiles#139]
We get a weird error when running `docker-compose run export` in Travis.

```bash
node: symbol lookup error: /usr/local/lib/node_modules/mapnik/lib/binding/node-v11-linux-x64/mapnik/input/postgis.input: undefined symbol: _ZN6icu_5513UnicodeStringC1ERKS0_
```

**Comment by lukasmartinelli** [comment]
It works in the `feature/detect-dirty-tiles` branch though.. :thought_balloon:

**Comment by manuelroth** [comment]
Seeing this error also in the `feature/detect-dirty-tiles` branch, it occurs in the module mapnik. I will merge the `feature/detect-dirty-tiles` branch anyways. We have to fix this in a later pull request. Maybe it helps to upgrade the mapnik node module to the latest version. But this is a task in its own.
![bildschirmfoto 2016-03-02 um 14 54 11](https://cloud.githubusercontent.com/assets/1810384/13462232/a7379358-e086-11e5-86b5-580b5150e91d.png)

**Comment by manuelroth** [comment]
@ImreSamu fixed this issue with his pull request #160. Really appreciate your contribution, thank you.
